packages/react-core/src/components/Form/__tests__/FormGroup.test.tsx

```diff
@@ -203,4 +204,70 @@ describe('FormGroup', () => {
     );
     expect(asFragment()).toMatchSnapshot();
   });
+
+  test('input should receive focus when clicking label', () => {
+    render(
+      <FormGroup data-testId="form-group-test-id" label="label" fieldId="native-label-element">
+        <input id="native-label-element" />
+      </FormGroup>
+    );
+
+    const labelElement = screen.getByTestId('form-group-test-id').querySelector('.pf-c-form__label');
+    const input = screen.getByTestId('form-group-test-id').querySelector('input');
```

Comment:
To better replicate the real user experience I would like if we could get away from relying on test ids to select these elements if possible.

I haven't had the time to test this myself yet to confirm, but I think that this should be possible via something like:

```suggestion
    const labelElement = screen.getByText('label');
    const input = screen.getByRole('textbox');
```

Reply:
Ah good catch! The `getByText` is more obvious in hindsight, but didn't think of the `textbox` role 🔥